CONTRIBUTING.md

```diff
@@ -0,0 +1,291 @@
+- We utilize automated code checks to verify that pull requests align with our established standards. These checks must be successful for the pull request to be merged into the main branch. You don't need to worry about this during development.
+  - The pull request title must adhere to the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard.
+  - We run linting and formatting checks on all the code.
+- When you are done with development you can mark the pull request as ready for review by clicking on the button at the bottom. A person from the design system team will then review your code and comment if there are things that need to be changed. Once the pull request is approved it will be merged into the main branch.
+
+## Styleguides
+
+### Commit Messages
+
+This project uses Lerna with the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
+specification in order to create nice and readable changelogs.
+
+The [Semantic Versioning 2.0](https://semver.org/) specification is used for versioning.
+
+To include commits in the changelog, please ensure that you include the following keywords:
+
+- Start the commit with `fix:` to trigger a patch (0.0.x) version.
+- Start the commit with `feat:` to trigger a minor (0.x.0) version.
+- Start the description / footer of a commit with `BREAKING-CHANGE:` to trigger a major (x.0.0) version. You also have to add either `fix:` or `feat:` to the main body of the commit when using `BREAKING-CHANGE:`. Do this when the changes directly effect the built files / components used by the end user.
+  See the examples further below to learn how to use the correct syntax.
+
+#### Scope
+
+To make commit messages and the changelog more specific and readable, you have the option to scope your commits by adding a keyword in parentheses that indicates the area or aspect you are working on. This practice helps provide clearer context and organization to the commit history.
+
+##### Examples:
+
+- Adding a new component: `feat(Button): added a new Button component`.
+- Adding documentation for icons: `docs(icons): added new documentation for the icons package`.
+
+#### When to use what keywords
+
+It is crucial to understand the distinctions between the two sections mentioned below. If you wish for commit messages to be included in the changelog, please use `fix:`, `feat:`, or `BREAKING-CHANGE:` as keywords. These keywords indicate changes that impact the users of our NPM packages and are therefore significant to highlight. For any other types of changes that do not directly affect the end user, please utilize a different keyword. If you are uncertain about which keyword to use and the changes are non-user-facing, you can use `chore:` as a default keyword.
```

Comment:
I don't think `BREAKING-CHANGE:` is a valid prefix, since it does not work in the body either.
Can we confirm somewhere?

Reply:
Maybe just remove it completety from the documentation for now?

Reply:
Unless we can confirm that it is working, I think we should. Last time we tried it had no effect

Reply:
agreed, just remove it